### ISSUE TITLE
Bad Request test for request with a body does not generate nested objects

### ISSUE THREAD

**Issue by TomMalow** [atc-net/atc#25]
**Describe the bug**
When defining a post endpoint that expects a body with nested objects, then the generated bad request test does not generate these nested object but instead places a `NEW-INSTANCE` tag. Version 208 placed a `new <object>()` instead, which is also invalid.

See example below
![image](https://user-images.githubusercontent.com/13435947/98686035-61d74180-2368-11eb-8d1d-a3c89525f7c1.png)

The above test should fail because `IdTag` is required but in this instance the test also fails because `NEW-INSTANCE` is not valid either.

**Expected behavior**
I expect the test to also create data for the nested object. 
However, I am unsure in the case of optional nested object if defining the object as null should be fine. 
Additional, should the generated bad request test for the nested objects?

**Environment:**
- Version 1.0.216.0

**Comment by davidkallesen** [comment]
Please verify that the bub is fixed from version 1.0.229

**Comment by TomMalow** [comment]
I have tested with 1.0.300 and it is not complete yet
Here is an example
![image](https://user-images.githubusercontent.com/13435947/98959546-aba85f00-2503-11eb-914c-0c1b9212f14e.png)

Problems:
- Indentation, minor issue
- Variable name for a nested object is placed both before and after the colon symbol. Example from above, line 77: `ChargingProfile : ChargingProfile` 

Additionally I found another unrelated bug. In line 75 we have the parameter `ConnectorId` and that is an integer but the test writes it as if it is a string. Will probably see if I can resolve that myself
